doc/articles/features/custom-fonts.md

```diff
@@ -0,0 +1,127 @@
+#### Custom Fonts on Android
```

Comment:
You should add a title to this file, and remove few "#" for this sub-title :)

Reply:
Fix'd :-)

Reply:
Thanks a lot for the quick approve!

Reply:
You're welcome :) I've put the ready-to-merge flag, but I don't know if it works with external contributor, otherwise I'll try to merge it later tonight.